lib/toxiproxy/version.rb

```diff
@@ -1,3 +1,3 @@
 class Toxiproxy
-  VERSION = "1.0.3"
+  VERSION = "1.0.4"
```

Comment:
No need to bump the version here, we usually bump the version, generate a new tag, and push a new release to rubygem seperately.